restler/engine/fuzzing_parameters/fuzzing_utils.py

```diff
@@ -151,16 +151,31 @@ def get_response_body(response):
     @rtype:  JSON
 
     """
+    from engine.transport_layer.messaging import DELIM
     # sanity cleanup
     response = response.replace('\\"', "'")
     response = response.replace("\\'", "'")
     response = response.replace("\'", "'")
 
     # extract the response body
-    body_start = response.find('{')
-    body_end = response.rfind('}')
+    body_start = response.find(DELIM)
+    if body_start == -1:
+        return None
+
+    try:
+        start_char = response[body_start + len(DELIM)]
+    except Exception:
```

Comment:
A short comment with an example would be useful (in particular, what DELIM vs. body_start are)